crates/bevy_render/src/primitives/mod.rs

```diff
@@ -502,4 +548,45 @@ mod tests {
             Aabb::from_min_max(Vec3::new(-1.0, -5.0, 0.0), Vec3::new(2.0, 0.0, 1.0))
         );
     }
+
+    // A frustum with an offset for testing the [`Frustum::contains_aabb`] algorithm.
+    fn contains_aabb_test_frustum() -> Frustum {
+        let proj = PerspectiveProjection {
+            fov: 90.0_f32.to_radians(),
+            aspect_ratio: 1.0,
+            near: 1.0,
+            far: 100.0,
+        };
+        proj.compute_frustum(&GlobalTransform::from_translation(Vec3::new(2.0, 2.0, 0.0)))
+    }
+
+    #[test]
+    fn aabb_inside_frustum() {
```

Comment:
All of these tests use Affine3A::IDENTITY, which is the most simplistic test case, and can hide issues regarding the Affine computations.

I'd recommend adding more tests with with different Affine transformations.